source/_posts/2020-11-16-short-python-screencasts-for-$50-slash-year.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Short Python screencasts for $50/year"
+date: 2020-11-16 15:26:13 -0800
+comments: true
+categories: python
+---
+
+I'm really excited to announce that Python Morsels subscriptions will now include concise screencasts.
+
+I will now be publishing at least one new Python screencast to active subscribers every week.
+
+These will be short, single-topic screencasts that won't waste your time.
+Each screencast will be at most 5 minutes in length and each will be as succinct as I can make it.
+
+These screencasts will be included in all current plans (both Lite and Standard).
+I'm also announcing a new $50/year version of the Lite plan (17% savings over the monthly plan).
+
+
+### What's included?
+
+There are two things included in this subscription: **weekly screencasts** and **monthly exercises**.
+
+You'll get *at least* one new screencast every week.
+If you enjoy the articles on my blog or the teaching style I use in my PyCon tutorials, you'll probably enjoy the format I use in my screencasts.
+I'm attempting to keep each screencast as narrowly focused and as concise as possible.
+
+If you don't like video, that's okay!
+If you prefer to watch but not listen, each screencast also includes English captions.
+And if you prefer text over video, you can read the **mini-blog post that's included with each screencast**, which acts as nearly a text-based equivalent to the video.
+
+In addition to the 1-2 succinct Python screencasts each week, you'll also receive **one Python exercise each month**.
+If you've ever worked through one of my PyCon tutorials, or attended a training session from me, you'll know that the best part of my curriculum is the exercises.
+I spend a *lot* of time coming up with Python exercises that are interesting and complex **without being overly complicated**.
+You don't need to work through the monthly exercises, but I recommend it.
+You don't learn by putting information into your head; you learn by trying to retrieve information from your head, and writing code is a great way to practice informational retrieval.
+
+
+### What are the screencasts like?
+
+Each screencast will be **concise** and **short**.
+Here are two examples: [variables are pointers][] (2 minutes) and [the 2 types of "change"][] (3 minutes).
+You can find more example screencasts on the Python Morsels [screencasts page][].
+
+{% img /images/screencasts.png %}
+
+**What topics will the screencasts be on?**
+Functions, classes, scope, operator overloading, decorators, exception handling, and lots more.
+The topics will range from beginner to advanced.
+
+**How many screencasts will be freely shareable?**
+I don't know yet.
```

Comment:
I am not sure but can we go without this sentence, _I think_  you're covering the points quite clearly in the last sentence of this section.